webrtc-stats.html

```diff
@@ -2005,6 +2006,15 @@ <h2>
                   Since it is only given for active tracks, the incremental exposure is small.
                 </p>
               </dd>
+              <dt>
+                <dfn>active</dfn> of type <span class=
+                "idlMemberType">boolean</span>
+              </dt>
+              <dd>
+                <p>
+                  Indicates whether this <a>RTP stream</a> is configured to be sent or disabled.
```

Comment:
Great. Can you just add another sentence like this or similar?
"An RTP stream that is configured to be sent can in some situations still be prevented from sending, such as when being CPU or bandwidth limited."

Reply:
Yes I agree with that I just want to clarify that "active" does not flip flop to false if we are limited

Reply:
Just like getParameters()'s active does not return false during BW limitation

Reply:
Done, mostly since i'm not sure that `limited` is easy to implement